docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
+Auf der Startseite wurden auch die NFC-Tags entsprechend der zugeordneten Gruppe gruppiert.
+Jegliche Neuerung auf dem Master wurden nun auch auf Github.io veröffentlicht.
+\subsubsection{Retrospektive}
+Deutlich besser lief in diesem Sprint die Problembewältigung durch Pair Programming, welches weiter die Teamarbeit und gegenseitige Hilfe verbesserte.
+Auch der Workflow mit Github, den Pull Request sowie Code Reviews wurde mehr verinnerlicht.
+Das Team lernte in diesem Sprint vor allem Vue Hooks kennen, und die Stärken von Feathers in Verbindung mit OAuth.
+Weiterhin waren die Tickets stärker voneinander abhängig, weshalb manche Arbeiten nicht direkt möglich machte.
+Um zukünftige Probleme gar nicht erst entstehen zu lassen, hatte sich das Team vorgenommen, entstandene Probleme nicht erst im Standup, sondern früher zu kommunizieren.
+Genauso sollte neu Gelerntes besser kommuniziert werden, um die Arbeit effizienter zu gestalten.
+
+\subsection{Meilenstein 4}
+Meilenstein vier lag innerhalb der Weihnachtszeit und wurde deswegen mit einem größeren Zeitraum vom 17.12.2020 bis 14.01.2021 geplant.
+Mit diesem Meilenstein begann die funktionale und visuelle Umsetzung der Hauptfunktionen.
+\subsubsection{Ziel}
+Wie auch beim vorherigen Meilenstein war wieder hier der Fokus auf den Kundenwunsch, dass die ersten Funktionalitäten umgesetzt werden.
+Dass umfasste das Anlegen von eigenen Tags, also dem Scannen von Tags, vergeben eines eigenen Namens, auswählen von Musik und einem eigenen Bild, sowie das bearbeiten bereits angelegter Tags.
+Auch sollten für den nächsten Schritt die Mockups erstellt werden, wie ein Nutzer den Lautsprecher zur Wiedergabe auswählt sowie seine eigene \textit{Leek Box} initial einrichtet.
+\subsubsection{Probleme}
+Zu diesem Meilenstein sind vermehrt größere Probleme aufgetreten.
+Die Planung und Einschätzung der umsetzbaren Tickets wurde schlecht eingeschätzt.
+Es wurde sich zu sehr in Details verloren, sodass der Fokus auf die wichtigen Tickets verloren ging.
+Das führte dazu, dass am Ende viele Tickets halbfertig und/oder nur schlecht getestet implementiert waren.
+\subsubsection{Lösungen}
+Kurzfristig wurde eine Team interne Evaluation der einzelnen Mitglieder durchgeführt.
+Das half einerseits dabei, dass Probleme gelöst und die Teammotivation sowie die Produktivität stieg, als auch das generelle Teamklima weiter gebessert wurde.
+\subsubsection{Product Increment}
+Das Produkt wurde um die Grundfunktionen erweitert.
+Es ist nun möglich, Tags anzulegen und entsprechend für diese Namen, Musik und ein Bild einzustellen.
+Genauso ist es dem Nutzer nun möglich, alle angelegten Tags zu durchsuchen.
+Es wurde weiterhin auch das Design Mockup von dem von Macio gestellten Designer in Vue größtenteils umgesetzt.
+Gleichzeitig wurden hierbei auch für eine leichtere Weiterarbeit jegliche Komponenten als möglichst allgemeine Vue Komponenten angelegt, um ein einheitliches Design zu ermöglichen.
```

Comment:
```suggestion
Gleichzeitig wurden hierbei die Komponenten zur einfacheren Verwendung als möglichst allgemeine Vue Komponenten angelegt, um ein einheitliches Design zu ermöglichen.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee